### PR TITLE
Update-clarify-sq-cmd

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr clarifies the resource type flags accepted for saved query. 

the --resource flag page  doesn't support`dbt run` so this pr also removes `dbt run` from the current page. 

[slack thread](https://dbt-labs.slack.com/archives/C03KHQRQUBX/p1715580715845999)

### CODE REVIEW

File: website/docs/reference/global-configs/resource-type.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ id: "resource-type"
 sidebar: "resource type"
 ---
 
-The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types (such as unit tests) from the `dbt run`, `dbt build`, and `dbt clone` commands.
+The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types (such as [saved queries](/docs/build/saved-queries) and [unit tests](/docs/build/unit-tests)) from the `dbt build` and `dbt clone` commands.
```
Comment (b-per) [inline website/docs/reference/global-configs/resource-type.md]: Is there a reason why we would list only those options?

Also, shouldn't we make it depend on the dbt version? (like unit tests don't exist before 1.8) ; or add some text saying that different dbt versions support different resource types?

**Review by b-per — CHANGES_REQUESTED** [review]

File: website/docs/reference/global-configs/resource-type.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/reference/global-configs/resource-type.md]: @b-per great flag! i can add a version block for 1.8+. is there a way i can check what types of flags are available for --resource-type? i tried checking in the cloud cli but i wasn't sure how to get a correct list

File: website/docs/reference/global-configs/resource-type.md [hunk 2]
```diff
@@ -4,13 +4,44 @@ id: "resource-type"
...
+</File>
+
+<VersionBlock firsttVersion="1.8" >
+
+In this example, run the following command to build `saved_queries` with the `--resource-type` flag:
 
+<File name='Usage'>
+
+```text
+dbt build --resource-type saved_query_name
```
Comment (mirnawong1) [inline website/docs/reference/global-configs/resource-type.md]:
```suggestion
dbt build --resource-type SAVED_QUERY_NAME
```

File: website/docs/reference/global-configs/resource-type.md [hunk 3]
```diff
@@ -4,13 +4,44 @@ id: "resource-type"
 sidebar: "resource type"
 ---
 
-The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types (such as unit tests) from the `dbt run`, `dbt build`, and `dbt clone` commands.
+The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types (such as unit tests) from the `dbt build` and `dbt clone` commands.
+
+The available resource types are:
+
+<VersionBlock lastVersion="1.7" >
+
```
Comment (mirnawong1) [inline website/docs/reference/global-configs/resource-type.md:23]: will add avail types in 1.7 and lower

File: website/docs/reference/global-configs/resource-type.md [hunk 4]
```diff
@@ -4,13 +4,59 @@ id: "resource-type"
...
+- [saved queries](/docs/build/saved-queries)  
+- [unit tests](/docs/build/unit-tests)
+
+</VersionBlock>
+
+## Example
+
+<VersionBlock lastVersion="1.7">
+
+For example, use the following command to exclude certain resources, such as unit tests, from your dbt build process:
```
Comment (mirnawong1) [inline website/docs/reference/global-configs/resource-type.md]:
```suggestion
For example, use the following command to exclude certain resources from your dbt build process: 
```

File: website/docs/reference/global-configs/resource-type.md [hunk 5]
```diff
@@ -4,13 +4,59 @@ id: "resource-type"
...
+
+<VersionBlock lastVersion="1.7">
+
+For example, use the following command to exclude certain resources from your dbt build process: 
 
 <File name='Usage'>
 
 ```text
-$ dbt build --exclude-resource-type UNIT_TEST_NAME
+$ dbt build --exclude-resource-type RESOURCE_TYPE
```
Comment (mirnawong1) [inline website/docs/reference/global-configs/resource-type.md]: add specific resource type available in 1.7 and lower

File: website/docs/reference/global-configs/resource-type.md [hunk 6]
```diff
@@ -4,13 +4,59 @@ id: "resource-type"
...
+  <File name='Usage'>
+
+```text
+dbt build --resource-type SAVED_QUERY_NAME
+```
+
+</File>
+
+
+- Use the following command to exclude certain resources, such as unit tests, from your dbt build process:
```
Comment (mirnawong1) [inline website/docs/reference/global-configs/resource-type.md]:
```suggestion
-  In this second example, use the following command to exclude certain resources, such as unit tests, from your dbt build process: 
```

File: website/docs/reference/global-configs/resource-type.md [hunk 7]
```diff
@@ -4,13 +4,67 @@ id: "resource-type"
...
+
+The available resource types are:
+
+<VersionBlock lastVersion="1.6">
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.7">
+
+- [saved queries](/docs/build/saved-queries)
```
Comment (mirnawong1) [inline website/docs/reference/global-configs/resource-type.md]: saved queries available in 1.7 and higher

File: website/docs/reference/global-configs/resource-type.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/reference/global-configs/resource-type.md]: ok i've added versions, huge thanks!

File: website/docs/docs/use-dbt-semantic-layer/exports.md [hunk 8]
```diff
@@ -241,7 +241,7 @@ You can use exports to create a custom integration with tools such as PowerBI, a
 
 <detailsToggle alt_header="How can I select saved_queries by their resource type?">
 
-To select `saved_queries` by resource type, run `dbt build --resource-type saved_query`.
+To build `saved_queries`, use the [`--resource-type` flag](/reference/global-configs/resource-type) and run the command `dbt build --resource-type saved_query_name`.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]:
```suggestion
To build `saved_queries`, use the [`--resource-type` flag](/reference/global-configs/resource-type), and run the command `dbt build --resource-type saved_query_name`.
```

**Comment by mirnawong1** [comment]
hey @b-per - going to come back to this tomorrow to finalize. can you let me know how i can find out what resources supports the resource type flag? i tried looking using --help flag but couldn't narrow it down

**Comment by b-per** [comment]
Yes, you can use `--help`.

But you will need to do `dbt run --help`, `dbt build --help` etc... to see the parameters for each subcommand.

Or you could check [this file in the source code ](https://github.com/dbt-labs/dbt-core/blob/751139d8c14503dfe1fa74295502e07c3b21b3d1/core/dbt/cli/main.py#L174-L175)to see which command supports it as well.

File: website/docs/docs/build/saved-queries.md [hunk 9]
```diff
@@ -214,6 +214,19 @@ Once you've configured exports, you can now take things a step further by runnin
...
+## FAQs
+
+<detailsToggle alt_header="Can I have multiple exports in a single saved query?">
+
+Yes, this is possible. However, the difference would be the name, schema, and materialization strategy of the export.
+</detailsToggle>
+
+<detailsToggle alt_header="How can I select saved_queries by their resource type?">
+
+To build `saved_queries`, use the [`--resource-type` flag](/reference/global-configs/resource-type) and run the command `dbt build --resource-type saved_query_name`.
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]:
```suggestion
To include all saved queries in the dbt build run, use the [`--resource-type` flag](/reference/global-configs/resource-type) and run the command `dbt build --resource-type saved_query`.
```

File: website/docs/docs/use-dbt-semantic-layer/exports.md [hunk 10]
```diff
@@ -198,7 +198,7 @@ You can use exports to create a custom integration with tools such as PowerBI, a
 
 <detailsToggle alt_header="How can I select saved_queries by their resource type?">
 
-To select `saved_queries` by resource type, run `dbt build --resource-type saved_query`.
+To build `saved_queries`, use the [`--resource-type` flag](/reference/global-configs/resource-type), and run the command `dbt build --resource-type saved_query_name`.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]:
```suggestion
To include all saved queries in the dbt build run, use the [`--resource-type` flag](/reference/global-configs/resource-type) and run the command `dbt build --resource-type saved_query`.
```

**Comment by b-per** [comment]
Is it normal that some words in [this page](https://docs-getdbt-com-git-update-clarify-sq-cmd-dbt-labs.vercel.app/reference/global-configs/resource-type) are singular and some of them plural?

**Comment by mirnawong1** [comment]
hey @b-per , great flag! thank you and i was going by the page configs but not the core set up. I've updated it now to the core source code-  when you have a sec, can you rereview! ty!

**Review by b-per — APPROVED** [review]

**Comment by mirnawong1** [comment]
thanks @b-per - always appreciate your insight! ✨

**Pull request merged**